apps/pxweb2/src/app/components/Presentation/Presentation.tsx

```diff
@@ -0,0 +1,96 @@
+import { useTranslation } from 'react-i18next';
+import { useEffect, useState} from 'react';
+
+import styles from './Presentation.module.scss';
+import { ContentTop } from '../ContentTop/ContentTop';
+import { Table, EmptyState } from '@pxweb2/pxweb2-ui';
+import useTableData from '../../context/useTableData';
+import useVariables from '../../context/useVariables';
+
+
+type propsType = {
+  selectedNavigationView: string;
+  selectedTabId: string;
+};
+
+export function Presentation({
+  selectedNavigationView,
+  selectedTabId,
+}: propsType) {
+  const { i18n, t } = useTranslation();
+  const tableData = useTableData();
+  const variables = useVariables();
+  const { pxTableMetadata } = useVariables();
+  const { hasLoadedDefaultSelection } = useVariables();
+  const { isLoadingMetadata } = useVariables();
+  const tableId: string = selectedTabId;
+  const [isMissingMandatoryVariables, setIsMissingMandatoryVariables] =
+    useState(false);
+  const [initialRun, SetInitialRun] = useState(true);
+  const { selectedVBValues } = useVariables();
+
+  useEffect(() => {
+    const hasSelectedValues = variables.getNumberOfSelectedValues() > 0;
+    // const SelectedValues = variables.getNumberOfSelectedValues();
+
+    const hasSelectedMandatoryVariables = pxTableMetadata?.variables
+      .filter((variable) => variable.mandatory)
+      .every((variable) =>
+        selectedVBValues.some(
+          (selectedVariable) => selectedVariable.id === variable.id
+        )
+      );
+    if (initialRun && !hasSelectedValues) {
```

Comment:
Maybe add comments about that we will replace this with dataloader?

Reply:
Comment added